ui/src/components/SelectImportPage/index.tsx

```diff
@@ -30,6 +32,7 @@ export const SelectImportPage = () => {
   const { getConnectedInfo, features } = useAppContext();
   const { setTotalSelectedRepos, setIsImportInProgress, setImportedRepositories } = useImportContext();
   const componentTypesResult = useComponentTypes();
+  const { isTeamsDataLoading, teams } = useTeamsForImport();
```

Comment:
nit: we can pass the object without destructuring it to make fewer props passing on the way to Select component